### DISCUSSION TITLE
How can I reload ca.json in docker

### DISCUSSION THREAD

**Discussion by JiangP95** [smallstep/certificates#782] [Q&A] [answered]
I run a  step-ca server in docker , then I change the ca.json , it doesn't work  Unless  I exit step-ca container and restart container.
how can i reload ca.json  without restart restart container.
can I restart step-ca in  container ? Or can I   "step-ca relaod ca.json "  in container

**Comment by maraino** [comment, marked as answer]
Hi @JiangP95, if you exec in the container and HUP the process it will reload the configuration, the PID is usually 1, so this should do it `kill -HUP 1`.

**Comment by JiangP95** [comment]
Hi @Mariano Cano


It worked. Thank you very much ! 


Best Wish 




On 12/18/2021 04:00，Mariano ***@***.***> wrote：

Hi @JiangP95, if you exec in the container and HUP the process it will reload the configuration, the PID is usually 1, so this should do it kill -HUP 1.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
